src/main/java/org/openstreetmap/atlas/geography/atlas/items/ConnectedEntityType.java

```diff
@@ -0,0 +1,19 @@
+package org.openstreetmap.atlas.geography.atlas.items;
+
+import java.util.function.Function;
+
+/**
+ * An interface that generalizes enum based declarative connectivity information.
+ *
+ * @param <ME>
```

Comment:
Sonar checks will flag this one. Change it to single letter names: `<M>`, and `<C>`

Reply:
Ok will update. Thanks.